### ISSUE TITLE
POE case refinement

### ISSUE THREAD

**Issue by bernardsilenou** [SORMAS-Foundation/SORMAS-Project#2897]
### Bug Description
- This is continuation of issue #2819
- If case is a POE case, place of stay is not optional. Cases mistakenly created as POE cases can not be corrected
### Steps to Reproduce
1.
2.

### Expected Behavior
- POE cases must not have a place of stay (facility or home) associated to them. It should be optional to add place of stay + facility to POE cased during case creating
- Should be possible to change the case origin after the POE case has been created
- Should be possible to change the case infrastructure data (region, district, community, facility) after the POE case has been created

### Screenshots
- 
![image](https://user-images.githubusercontent.com/12996724/93370579-54627700-f851-11ea-9b24-222755f491ff.png)

### System Details
* Device: web
* SORMAS version: 1,48,1
* Android version/Browser:

### Additional Information
- Another option is to just hide the place of stay on the case creation form for POE cases

**Comment by bernardsilenou** [comment]
@MateStrysewske , @Iheanacho2027 What is your suggestion?

**Comment by Iheanacho2027** [comment]
I agree with most of your suggestions @bernardsilenou except that place of stay should be completely disabled and shouldnt even be made optional when reporting POE cases, based on the fact that the form displayed above is concerned with where the case was reported from. Place of stay can be brought up when the case is being transferred in-country. Thanks

**Comment by MateStrysewske** [comment]
@Iheanacho2027 That is the case - the place of stay can only be edited when the case was transferred. Still, we need to take care of this. It's also possible that the app has the same bug, so we should check that, too.

**Comment by MateStrysewske** [comment]
Seems like the bug is already resolved. The refinement will be taken care of later as we have more pressing priorities right now.

**Comment by bernardsilenou** [comment]
@MateStrysewske  A GE user created a POE case mistakenly and want to change it.  Can we get this back on board soon? any further refinement you can identify?

**Comment by MateStrysewske** [comment]
@bernardsilenou Depends on priorities. For now, I'd suggest to simply delete the case if something like this happens and create a new one with the correct details.
